.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Contact Tenzir
+    url: https://tenzir.com/contact
+    about: Reach out to Tenzir
+  - name: Gitter Chat
```

```suggestion
  - name: Chat
```

Is it important that this is Gitter? Maybe it'll change in the future.